test/sagemaker_tests/huggingface_pytorch/training/integration/sagemaker/test_smdp.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+import sagemaker.huggingface
+from sagemaker.huggingface import HuggingFace
+
+from test.test_utils import get_framework_and_version_from_tag, get_cuda_version_from_tag
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+from ...integration import DEFAULT_TIMEOUT
+from ...integration.sagemaker.timeout import timeout
+import sagemaker
+
+git_config = {'repo': 'https://github.com/huggingface/transformers.git', 'branch': 'v4.4.2'}
```

Comment:
Does this mean we need to update the `branch` on each transformer version bump? Can we declare this universally so that other test that refer it?

Reply:
Thanks for the comment. I will update it to read it from image tag / master in the next PR. Creating an issue to track this.